### PR TITLE
feat: add VHDL support

### PR DESCRIPTION
**Pull request by Mrcuve0**
Hi!
I added some snippets supporting the VHDL language. The snippets were ported from https://github.com/richjyoung/vscode-modern-vhdl/blob/master/snippets/vhdl.yml

I tested many of them and they do work as intented

### CODE REVIEW

**Comment by Mrcuve0** [comment]
I'm now testing the snippets from TerosHDL, another plugin from VSCode. As for now, it has less users than modern-vhdl (30k v. 50k) but I see it becoming way more popular over time (there's just no comparison in the functionalities provided, Teros does so much more).

The goal is to select the best snippets from one of the two plugins, taking into account those that may become "de-facto standard" in a near future. 

As for now, TerosHDL has way more snippets, however I'm having some unexpected behavior with some of them, I may need help understanding why

Do not merge as for now

**Comment by Mrcuve0** [comment]
A snippet that is giving me issues on neovim is the one triggered by the keyword "useieee".
The snippets allows to select, using completion, some of the most common libraries, see screenshot below.

image

This behavior does not happen in neovim, nvim-cmp is not triggered. I don't know if the issue is related to the snippet sintax which goes out-of-standard or to nvim-cmp. See next two screenshots:

Screenshot from 2023-01-20 12-18-32

Expanding the snippet accepts automatically the first proposal but does not trigger nvim-cmp to accept all the remaining ones. Jumping with tab simply puts the cursor on the next line. Manually invoking cmp works but does not shows the snippet's proposals.

Screenshot from 2023-01-20 12-18-47

**Comment by OkelleyDevelopment** [comment]
Hey @Mrcuve0 is this PR ready for review or are you still making edits? Just let me know and then we can get someone to review it :+1:

**Comment by Mrcuve0** [comment]
Hi! Sorry had a busy week and forgot to update, I hadn't time to test everything but many of these snippets seems really good! IMO we can merge it

**Comment by OkelleyDevelopment** [comment]
No worries! 

I'll give it another read through just to confirm in a bit (or tomorrow evening, whichever happens first lol) and then will merge if its all good. :sunglasses:

**Review by OkelleyDevelopment — APPROVED** [review]
Granted I'm not super familiar with VHDL, but upon a general review these look like they should be useful :sunglasses:

**Comment by Mrcuve0** [comment]
Hi!

Sorry for the trouble, unfortunately I just now realized that I forgot to update the snippets file with the version taken from TerosHDL. Instead, the currently commited version came from a first plugin I had tested, which however has really fewer snippets when compared to the TerosHDL snippets ([which you can find here](https://github.com/TerosTechnology/vscode-terosHDL/blob/main/snippets/vhdl/vhdl.json). I tested this version as it is, without any further modification to the file)

I am really sorry, it was a really stupid distraction and I was convinced I had commited the final version of the file. Is it possible to update the snippets file with [this one](https://github.com/TerosTechnology/vscode-terosHDL/blob/main/snippets/vhdl/vhdl.json)?

**Pull request merged**
